ui/components/app/modals/account-nickname-modal/account-nickname-modal.container.js

```diff
@@ -0,0 +1,30 @@
+import { connect } from 'react-redux';
+import { hideModal, showModal } from '../../../../store/actions';
+import AccountNicknameModal from './account-nickname-modal.component';
+
+function mapStateToProps(state) {
```

Comment:
We no longer use this pattern for modals. Instead see `metamask-extension/ui/components/app/qr-hardware-popover/qr-hardware-popover.js` as an example: we conditionally render a `<Popover ... />` component, and so the properties can be directly passed from the parent to the popover.

Reply:
Fixed ! we have created a composable functional component responsible for rendering the appropriate nickname modal according to its internal state ! all other modals have been removed.